src/Http/Routing/src/Builder/EndpointRouteBuilderExtensions.cs

```diff
@@ -187,5 +189,324 @@ public static class EndpointRouteBuilderExtensions
 
             return dataSource.AddEndpointBuilder(builder);
         }
+
+
+        /// <summary>
+        /// Adds a <see cref="RouteEndpoint"/> to the <see cref="IEndpointRouteBuilder"/> that matches HTTP GET requests
+        /// for the specified pattern.
+        /// </summary>
+        /// <param name="endpoints">The <see cref="IEndpointRouteBuilder"/> to add the route to.</param>
+        /// <param name="pattern">The route pattern.</param>
+        /// <param name="handler">The delegate executed when the endpoint is matched.</param>
+        /// <returns>A <see cref="IEndpointConventionBuilder"/> that can be used to further customize the endpoint.</returns>
+        public static RouteHandlerBuilder MapGet(
+            this IEndpointRouteBuilder endpoints,
+            string pattern,
+            Delegate handler)
+        {
+            return MapMethods(endpoints, pattern, GetVerb, handler);
+        }
+
+        /// <summary>
+        /// Adds a <see cref="RouteEndpoint"/> to the <see cref="IEndpointRouteBuilder"/> that matches HTTP POST requests
+        /// for the specified pattern.
+        /// </summary>
+        /// <param name="endpoints">The <see cref="IEndpointRouteBuilder"/> to add the route to.</param>
+        /// <param name="pattern">The route pattern.</param>
+        /// <param name="handler">The delegate executed when the endpoint is matched.</param>
+        /// <returns>A <see cref="IEndpointConventionBuilder"/> that can be used to further customize the endpoint.</returns>
```

Comment:
Can we change the docs  from `IEndpointConventionBuilder` to `IEndpointRouteBuilder ` ? see other places as well.

Reply:
or we did not rename this `IEndpointConventionBuilder` ?

Reply:
We cannot rename `IEndpointConventionBuilder` because we started shipping that in 3.0.

Reply:
I can update the doc comments to refer to the `RouteHandlerBuilder` which implements `IEndpointConventionBuilder` instead though.

Reply:
Thanks. It is also a much bigger change.

Reply:
Updated.

Reply:
Much cleaner.. I don't have to search for `IEndpointConventionBuilder `